java/src/Ice/build.gradle

```diff
@@ -26,6 +26,5 @@ apply from: "$project.ext.topSrcDir/java/gradle/library.gradle"
 javadoc.dependsOn(compileJava)
 
 javadoc {
-    classpath = project.sourceSets.main.output
```

Comment:
Does this disable javadoc?

Reply:
No, javadocs still build fine. Before we were just restricting what files could be referenced in the docs. I'm planning on doing a more exhaustive comparison Monday, but from when I compared them, they seemed identical.
But this doesn't affect which classes we generate documentation for, just what those documentations are allowed to reference.